assets/wizards/analytics/views/configuration/index.js

```diff
@@ -0,0 +1,164 @@
+/* global newspack_analytics_wizard_data */
+
+/**
+ * WordPress dependencies
+ */
+import { Component, Fragment } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import {
+	Button,
+	Notice,
+	TextControl,
+	SelectControl,
+	CheckboxControl,
+	withWizardScreen,
+} from '../../../../components/src';
+import './style.scss';
+
+const SCOPES_OPTIONS = [
+	{ value: 'HIT', label: __( 'Hit', 'newspack' ) },
+	{ value: 'SESSION', label: __( 'Session', 'newspack' ) },
+	{ value: 'USER', label: __( 'User', 'newspack' ) },
+	{ value: 'PRODUCT', label: __( 'Product', 'newspack' ) },
+];
+
+/**
+ * Analytics Configuration screen.
+ */
+class Configuration extends Component {
+	state = {
+		error: newspack_analytics_wizard_data.customDimensions.error,
+		customDimensions: newspack_analytics_wizard_data.customDimensions,
+		newDimensionName: '',
+		newDimensionScope: SCOPES_OPTIONS[ 0 ].value,
+	};
+
+	handleAPIError = ( { message: error } ) => {
+		this.setState( { error } );
+	};
+
+	handleCustomDimensionCreation = () => {
+		const { wizardApiFetch } = this.props;
+		const { customDimensions, newDimensionName, newDimensionScope } = this.state;
+		wizardApiFetch( {
+			path: '/newspack/v1/wizard/analytics/custom-dimensions',
+			method: 'POST',
+			data: { name: newDimensionName, scope: newDimensionScope },
+		} )
+			.then( newCustomDimension => {
+				this.setState( { customDimensions: [ ...customDimensions, newCustomDimension ] } );
+			} )
+			.catch( this.handleAPIError );
+	};
+
+	handleCategoryDimensionSetting = dimensionId => {
+		const { wizardApiFetch } = this.props;
+		const { customDimensions } = this.state;
+		wizardApiFetch( {
+			path: `/newspack/v1/wizard/analytics/category-dimension/${ dimensionId }`,
+			method: 'POST',
+		} )
+			.then( ( { id } ) => {
+				this.setState( {
+					customDimensions: customDimensions.map( dimension => ( {
+						...dimension,
+						is_category_dimension: dimension.id === id,
+					} ) ),
+				} );
+			} )
+			.catch( this.handleAPIError );
+	};
+
+	render() {
+		const { error, customDimensions, newDimensionName, newDimensionScope } = this.state;
+		const hasCustomDimensions = ! error && customDimensions.length !== 0;
+		return (
+			<div className="newspack__analytics-configuration">
```

Comment:
The card is missing some padding at the bottom -- it looks like this combination of classes is normally used to add that:

`newspack-card newspack-card__no-background`

If these could be added to this element, or a div nested inside of it, the spacing should match other cards:

```suggestion
			<div className="newspack__analytics-configuration newspack-card newspack-card__no-background">
```